nexus/db-model/src/vpc_firewall_rule.rs

```diff
@@ -211,20 +211,55 @@ pub struct VpcFirewallRule {
     pub priority: VpcFirewallRulePriority,
 }
 
+/// Cap on the number of rules in a VPC
+///
+/// The choice of value is somewhat arbitrary, but the goal is to have a
+/// large number that customers are unlikely to actually hit, but which still
+/// meaningfully limits the ability to overload the DB with a single request.
+const MAX_FW_RULES_PER_VPC: usize = 1024;
+
+/// Cap on targets and on each type of filter
+const MAX_FW_RULE_PARTS: usize = 256;
```

Comment:
Just occurred to me. We don't support port ranges (e.g. "3000-4000") as a value, do we? If so, we should increase this number. Or better yet, add support for port ranges at some point.

If we do support port ranges, then ignore this comment :)

Reply:
We do support ranges. You just put in `“123-456”`.

Reply:
Sweet! Can we add that to the docs?

Reply:
It's sort of in there, but I will improve the wording.

<img width="672" alt="image" src="https://github.com/user-attachments/assets/1df4b117-0208-4747-995d-54213a90f6fe">